awx/main/models/events.py

```diff
@@ -557,6 +559,26 @@ def _update_host_summary_from_stats(self, hostnames):
                 summary = JobHostSummary(created=now(), modified=now(), job_id=job.id, host_id=host_id, host_name=host, **host_stats)
                 summary.failed = bool(summary.dark or summary.failures)
                 summaries[(host_id, host)] = summary
+                if self.job.inventory.kind == 'constructed':
+                    # make an extra copy of summary for original host
```

Comment:
As we discussed on our review call, we might look at adding an FK to the constructed host in the new row instead of having a copy of the row for original and constructed.